src/Components/Facility/ConsultationForm.tsx

```diff
@@ -641,6 +641,29 @@ export const ConsultationForm = ({ facilityId, patientId, id }: Props) => {
           }
           return;
         }
+        case "weight": {
+          if (state.form.suggestion !== "DD") {
+            const value = state.form[field];
+            if (!value || parseFloat(value) <= 0) {
+              errors[field] = "Please enter a valid Weight";
+              invalidForm = true;
+              break;
+            }
+          }
+          return;
+        }
+
+        case "height": {
+          if (state.form.suggestion !== "DD") {
+            const value = state.form[field];
+            if (!value || parseFloat(value) <= 0) {
+              errors[field] = "Please enter a valid Height";
+              invalidForm = true;
+              break;
+            }
+          }
+          return;
+        }
```

Comment:
```suggestion
        case "weight": 
        case "height": {
          if (state.form.suggestion !== "DD") {
            const value = state.form[field];
            if (!value || parseFloat(value) <= 0) {
              errors[field] = `Please enter a valid ${field}`;
              invalidForm = true;
              break;
            }
          }
          return;
        }
```

@Keshav-0907 The logic looks similar, why not combine the blocks, would be much easier to read. Please do the same in other places too